MsweatherRequest/src/main/java/com/example/MsweatherRequest/configuration/RabbitProducer.java

```diff
@@ -0,0 +1,26 @@
+package com.example.MsweatherRequest.configuration;
+
+
+import com.example.MsweatherRequest.model.WeatherRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class RabbitProducer {
+
+    @Autowired
+    RabbitTemplate rabbitTemplate;
+
+    public RabbitProducer(final RabbitTemplate rabbitTemplate) {
+        this.rabbitTemplate = rabbitTemplate;
+    }
+
+    public void SendToQueue(WeatherRequest weatherRequest ) {
```

Comment:
server doesn't need to know about api request object, use dto

Reply:
done